windows_install_build_dependencies.bat

```diff
@@ -508,6 +427,31 @@ IF /I EXIST %opt_install_dir%\%curl_dir% (
     if !errorlevel! neq 0 exit /b !errorlevel!
 )
 
+:: Create lib file for libgit2 linking
```

Comment:
this code looks strange to me. isn't is enough to add to PATH %BAZEL_VS and execute
lib.exe /def:C:\opt\curl-8.14.1_1-win64-mingw\bin\libcurl-x64.def /out:C:\opt\curl-8.14.1_1-win64-mingw\bin\libcurl-x64.lib

Reply:
The for loop is needed to find the SDK that is installed, we use different versions for jenkins node, laptops, and with new BuildTools release by MSC we would needto update this.

The path we look for is:
Matched directory: C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.28.29333
Where 14.28.29333 is the key and differs per platform.

for /d %%F in ("!CURL_LIB_PATH!") do (